.github/workflows/nf-test-gpu.yml

```diff
@@ -0,0 +1,154 @@
+name: Run nf-test on GPU
+on:
+  push:
+    branches:
+      # https://docs.renovatebot.com/key-concepts/automerge/#branch-vs-pr-automerging
+      - "renovate/**" # branches Renovate creates
```

Comment:
does renovate create branches in sarek?

Reply:
Indeed no, I'll clean that up in my TEMPLATE PR too